test/unit/sessions/video-session-handler.test.ts

```diff
@@ -1431,3 +1432,52 @@ a=msid:cbf2ec37-5e50-4ac4-9ae7-1d1dc4508071 ${trackId} `; /* should trim this bl
     expect(handler.getTrackIdFromSdp(sdp, 'audio')).toBe(trackId);
   });
 });
+
+describe('handleDataChannelMessage', () => {
+  it('should call handleMemberStatusMessage', () => {
+    const spy = handler.handleMemberStatusMessage = jest.fn();
+
+    const message: IMemberStatusMessage = {
+      jsonrpc: 'v2',
+      method: 'member.notify.status',
+      params: {
+        speakers: []
+      }
+    };
+
+    handler.handleDataChannelMessage({} as any, message);
+
+    expect(spy).toHaveBeenCalled();
+  });
+
+  it('should log a warning if there is no message handler', () => {
+    const spy = handler.handleMemberStatusMessage = jest.fn();
+    const logSpy = jest.spyOn(handler as any, 'log');
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/MyPureCloud/genesys-cloud-webrtc-sdk/pullRequest?prid=9993633)
```suggestion
    const logSpy = jest.spyOn(handler as unknown, 'log');
```